lib/solidus_social.rb

```diff
@@ -7,20 +7,37 @@
 require 'solidus_social/version'
 require 'solidus_social/engine'
 
+
 module SolidusSocial
-  def self.configured_providers
-    ::Spree::SocialConfig.providers.keys.map(&:to_s)
-  end
+  OAUTH_PROVIDERS = [
```

Comment:
Can you explain what's wrong with `configured_providers`? Why do we need to change it into a constant?

Reply:
Hi @kennyadsl Earlier, we used to configure the providers using an initializer file, where we configure provider with credentials, I find it would be more appropriate if we pull the provider and credentials details from the `:Spree::AuthenticationMethod` table instead, so I removed this method. 

Constant I took, just to maintain the list of provider into the extension itself. 
Let me know your thoughts on this.